src/rust/ide/src/lib.rs

```diff
@@ -152,3 +152,42 @@ pub fn run_ide() {
         project_view.forget();
     });
 }
+
+
+
+// =============
+// === Tests ===
+// =============
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+    use enso_protocol::project_manager::API;
+    use enso_protocol::project_manager::Client;
+
+    use wasm_bindgen_test::wasm_bindgen_test_configure;
+
+    wasm_bindgen_test_configure!(run_in_browser);
+
+
+    //#[wasm_bindgen_test::wasm_bindgen_test(async)]
```

Comment:
Have you checked if `#[ignore]` work with wasm_bindgen_test? I would be really surprised if it does not.

Reply:
@farmaazon yeah, I have tried it and it doesn't work.